.pipelines/ccf.yml

```diff
@@ -11,11 +11,24 @@ parameters:
 
 steps:
   - script: |
+      set -ex
+      sudo rm -rf $(Pipeline.Workspace)/CCF
       git clone --single-branch -b ccf-${{ parameters.Version }} https://github.com/microsoft/CCF $(Pipeline.Workspace)/CCF
     displayName: Checkout CCF ${{ parameters.Version }}
 
   - script: |
-      cmake -GNinja -B build \
+      set -ex
+      cd getting_started/setup_vm
+      ./run.sh ccf-dev.yml -e ccf_ver=${{ parameters.Version }} -e platform=${{ parameters.Platform }} -e clang_version=15
+    workingDirectory: $(Pipeline.Workspace)/CCF
+    displayName: Setup CCF development environment
+
+  - script: |
+      set -ex
+      sudo rm -rf build
```

Comment:
this directory should not exist as it is running on a fresh agent

Reply:
the agent might not be necessarily new, so I'd keep this

Reply:
I had the same thought. While debugging other issues, I noticed that CCF do the same in their build, so I thought it might not hurt too have it here too just in case.